src/poprox_concepts/domain/article.py

```diff
@@ -39,3 +39,13 @@ class ArticleSet(BaseModel):
     model_config = ConfigDict(extra="allow")
 
     articles: list[Article]
+
+
+class ScrapedArticle(BaseModel):
+    title: str
+    description: str | None = None
+    url: str
+    section: str | None = None
+    level: str | None = None
+    image_url: str | None = None
+    created_at: datetime = datetime.now(timezone.utc)
```

Comment:
We don't always include created_at in our model objects -- two reasons. First we don't really want this auto-assigning to when the model object was created like it does here -- the source of truth for that should probably be the database. But secondly, we just rarely need that information in python so we add it only when we know we need it.